Zotero/Scenes/General/Views/CollectionsPickerViewController.swift

```diff
@@ -124,16 +124,7 @@ class CollectionsPickerViewController: UICollectionViewController {
     }
 
     private func updateTitle(with selectedCount: Int) {
```

Comment:
This doesn't need to be a separate function anymore.

Reply:
It's called in 2 different places, so it's better if we keep it as is.